src/components/Dashboard/TodoList.vue

```diff
@@ -139,8 +154,15 @@
                   </div>
                   <div class="payment-status" v-else>
                     <span>PAID</span>
-                    <v-btn small icon color="black" class="info-btn">
-                      <v-icon color="red darken-4">mdi-information-outline</v-icon>
+                    <v-btn
+                      class="expand-btn"
+                      outlined
+                      color="red darken-4"
+                      :ripple=false
```

Comment:
Is ripple a little graphic when clicked?

Reply:
It's like of like a wave of colour starting from the middle and going to the edges. It's passe, frankly :)  And I'm just following the previous design.